src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -116,7 +116,7 @@ protected function isPropertyEnabled(string $property): bool
      */
     protected function isPropertyMapped(string $property, string $resourceClass, bool $allowAssociation = false): bool
     {
-        if ($this->isPropertyNested($property)) {
+        if ($this->isPropertyNested($property) && !$this->isPropertyNestedEmbedded($property, $resourceClass)) {
```

Comment:
@teohhanhui , @soyuka , to be sure beacuse I'm french :-D . When we add an embed field, we create a "nested" property access ? 
Example:
- Entity A contains Embed B -> we access like entity_a.entity_b.field. It is considered as a nested access ? (difficult to explain with words :-D)